### PR TITLE
Update servicemap processor to ExportTraceServiceRequest

### PR DESCRIPTION
**Pull request by kowshikn**
*Issue #58  :*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by AustinTag — APPROVED** [review]
LGTM

File: situp-plugins/service-map-stateful/src/test/java/com/amazon/situp/plugins/processor/ServiceMapStatefulProcessorTest.java [hunk 1]
```diff
@@ -3,12 +3,7 @@
 import com.amazon.situp.model.record.Record;
 import java.io.File;
 import java.time.Clock;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Objects;
-import java.util.Set;
+import java.util.*;
```
Comment (sshivanii) [inline situp-plugins/service-map-stateful/src/test/java/com/amazon/situp/plugins/processor/ServiceMapStatefulProcessorTest.java:6]: Nit: Bala suggested avoiding * imports, and preferring single class imports.

File: situp-plugins/service-map-stateful/src/test/java/com/amazon/situp/plugins/processor/ServiceMapTestUtils.java [hunk 2]
```diff
@@ -3,15 +3,13 @@
 import com.google.protobuf.ByteString;
 import com.amazon.situp.model.record.Record;
 import java.io.UnsupportedEncodingException;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.Random;
-import java.util.Set;
+import java.util.*;
```
Comment (sshivanii) [inline situp-plugins/service-map-stateful/src/test/java/com/amazon/situp/plugins/processor/ServiceMapTestUtils.java:6]: Same as above: Preferred to use single class imports

**Review by sshivanii — APPROVED** [review]
LGTM

**Pull request merged**